src/libsets.h

```diff
@@ -39,4 +39,5 @@ void Pop(SetElem **tail);
 void PrintAll(SetElem **tail);
 int Size(SetElem **tail);
 int GetElem(SetElem **tail, int position);
-int IsExist(SetElem **tail, int64_t num);
\ No newline at end of file
+int IsExist(SetElem **tail, int64_t num);
+int64_t PowerSetSize(SetElem **tail);
```

Comment:
Переименовать функцию
```suggestion
int64_t PowerSet(SetElem **tail);
```